toolkit/geoview-compose/src/main/java/com/arcgismaps/toolkit/geoviewcompose/SceneView.kt

```diff
@@ -345,6 +344,7 @@ public fun SceneView(
                         addAll(imageOverlays)
                     }
                 }
+                it.cameraController = cameraController
```

Comment:
Worth to add a comment here along the lines...
```suggestion
                // Set the camera controller last, to ensure other dependend SceneView properties are already set.
                // For example, OrbitGeoElementCameraController requires its associated GeoElement to be in a graphics overlay
                // set on the SceneView at this point.
                // https://devtopia.esri.com/runtime/kotlin/issues/6623
                it.cameraController = cameraController
```